hands-on-exercise/5-ibc-adv/6-ibc-app-steps.md

```diff
@@ -52,7 +52,7 @@ modify x/leaderboard/types/keys.go
 
 <HighlightBox type="warning">
 
-The code in this section was scaffolded with Ignite CLI v0.22. This includes ibc-go v3 as a dependency. The latest version of ibc-go is already past v3 so there may be some differences compared to the code in this section. For documentation on the latest version of ibc-go, please refer to the [ibc-go docs](https://ibc.cosmos.network/main/ibc/apps/apps.html).
+The code in this section was scaffolded with Ignite CLI v0.22. This includes ibc-go v3 as a dependency, which is [discontinued](https://github.com/cosmos/ibc-go/blob/143e5d85e9d2fc5fc2fc53adc42127a439ee2b79/RELEASES.md#stable-release-policy). The latest version of ibc-go is already [past v4](https://github.com/cosmos/ibc-go/blob/main/RELEASES.md#stable-release-policy) so there may be some differences compared to the code in this section. For documentation on the latest version of ibc-go, please refer to the [ibc-go docs](https://ibc.cosmos.network/main/ibc/apps/apps.html).
```

Comment:
```suggestion
The code in this section was scaffolded with Ignite CLI v0.22.1. This version includes ibc-go v3 as a dependency, which has [reached past end-of-life](https://github.com/cosmos/ibc-go/blob/143e5d85e9d2fc5fc2fc53adc42127a439ee2b79/RELEASES.md#stable-release-policy) and is no longer actively supported. 

**It is thus highly discouraged to deploy any code in production using ibc-go code scaffolded by Ignite CLI v0.22.1.**

<!--- TODO: add link to section on upgrading ibc-go version --->

[All actively supported version of ibc-go](https://github.com/cosmos/ibc-go/blob/main/RELEASES.md#stable-release-policy) have reached past v3, so there may be some differences compared to the code in this section. For documentation on the latest version of ibc-go, please refer to the [ibc-go docs](https://ibc.cosmos.network/main/ibc/apps/apps.html).
```

Reply:
@murataniloener do we want to wait to merge this until there's some content on upgrading one's (SDK or) ibc-go version? Or do we merge it already and make an issue to add it later?

Reply:
I think we should merge this. 
Later if we see how our upgrade guide looks like, we can decide what to do in this section.  
One should not use Ignite with the Dragonberry bug either so maybe we can include a link to https://github.com/ignite/cli/tree/main#cosmos-sdk-compatibility if we talk about production deployment. We can recommend to use latest version of Ignite for production but it still will not come with the latest SDK and IBC-go versions. So I think the upgrading section will be essential for production deployments.

Reply:
I agree